homeassistant/core.py

```diff
@@ -99,9 +100,10 @@
     from .config_entries import ConfigEntries
 
 
-STAGE_1_SHUTDOWN_TIMEOUT = 100
-STAGE_2_SHUTDOWN_TIMEOUT = 60
-STAGE_3_SHUTDOWN_TIMEOUT = 30
+STAGE_1_SHUTDOWN_TIMEOUT = 60
+STAGE_2_SHUTDOWN_TIMEOUT = 40
```

Comment:
Lowering the timeout for STOP is likely a breaking change as it mean integration only have 40 seconds to react to a stop event

Reply:
I'd rename these constants to match the corresponding name of the event since we now have a lot of stages

Reply:
You're right. Plus, unloading is more important than automations.

I think I should also change the log messages in case of timeouts, not just the constants. Thoughts?

Reply:
Would make sense to adjust the log messages as well

Also it's very important that the total amount of time in these constants doesn't increase as core could get killed by supervisor if it takes too long since it will think it's crashed

Reply:
Also any adjustment downward has to be a breaking change since it affects the maximum time an integration has to shutdown. That's probably ok though